bin/thumbnail

```diff
@@ -38,6 +38,7 @@ MODE_OPTIONS=(
 	"fixed-aspect-ratio"
 	"fixed-aspect-ratio-down"
 	"scale-to-width"
+	"scale-only-down"
```

Comment:
Technically, it is "scale down to width or height, whichever given". Any other naming proposal? What about simply "scale down"?